contracts/instruments/RibbonThetaVault.sol

```diff
@@ -281,83 +279,48 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
      * @param share is the number of vault shares to be burned
      */
     function withdraw(uint256 share) external nonReentrant {
-        uint256 withdrawAmount = _withdraw(share, false);
+        uint256 withdrawAmount = _withdraw(share);
         IERC20(asset).safeTransfer(msg.sender, withdrawAmount);
     }
 
     /**
      * @notice Burns vault shares and checks if eligible for withdrawal
      * @param share is the number of vault shares to be burned
-     * @param isScheduled is whether the withdraw was scheduled
      */
-    function _withdraw(uint256 share, bool isScheduled)
-        private
-        returns (uint256)
-    {
+    function _withdraw(uint256 share) private returns (uint256) {
         (uint256 amountAfterFee, uint256 feeAmount) =
             withdrawAmountWithShares(share);
 
         emit Withdraw(msg.sender, amountAfterFee, share, feeAmount);
 
-        _burn(isScheduled ? address(this) : msg.sender, share);
+        _burn(msg.sender, share);
 
         IERC20(asset).safeTransfer(feeRecipient, feeAmount);
 
         return amountAfterFee;
     }
 
     /**
-     * @notice Lock's users shares for future withdraw and ensures that the new short excludes the scheduled amount.
```

Comment:
why are we removing withdrawLater? what if somebody wants to just exit the vault, not necessarily migrate to v2

Reply:
A few reasons,
* we didn't have a good way around setting the user's pricePerShare, which we solved in v2
* we don't have frontend support for that in v1

Reply:
They can also just call the `withdraw` function normally

Reply:
We might want to support users who want to migrate during the rollover period, where 100% of the vault is eligible for withdraw. Maybe an additional prompt in the UI explaining that if they submit a tx during that period it'll work?